packages/nextra-theme/src/layout/MDXLayoutNav.tsx

```diff
@@ -58,7 +58,7 @@ const DesktopWrapper = ({ children }: PropsWithChildren<{}>) => {
 
 const MobileWrapper = ({ title, children }: PropsWithChildren<{ title?: string }>) => {
   const [open, setOpen] = useState(false)
-  const { t } = useI18n()
+  const { t } = useI18n<any>()
```

Comment:
Adding `any` is obviously not a great fix, but it's how `useI18n` is currently used [elsewhere in `nextra-theme`](https://github.com/graphprotocol/docs/blob/main/packages/nextra-theme/src/components/EditPageLink.tsx#L23). I still don't fully understand the point of extracting the theme to a separate package if I'm being completely honest; we wouldn't need to use `any` if this was in the `website` package because the translations live there.

Reply:
We extracted it to be possible to install the same theme in another repository? We can back nextra-theme in the `website` folder if it is no longer the case with remote mdx cc @saihaj